onnxruntime/contrib_ops/cuda/bert/attention.cc

```diff
@@ -89,6 +90,8 @@ Status Attention<T>::ComputeInternal(OpKernelContext* context) const {
   bool is_1d_mask = nullptr != mask_index && mask_index->Shape().NumDimensions() == 1;
 
   MHARunner* fused_runner = nullptr;
+
+#ifndef ENABLE_TRAINING  // Only enable fused kernel on non-training builds
```

Comment:
Walkaround for build error (`cl : command line error D8040: error creating or communicating with child process`) in Windows CPU CI Pipeline training_x64_release. So we disable it for training until we find a solution.